ibis/tests/all/test_client.py

```diff
@@ -14,6 +15,20 @@
 )
 
 
+def _drop(con, name, method_name, drop_kwargs, create_kwargs):
```

Comment:
instead of this, can you simply create a fixture which does this? it is not at all obvious *where* this is used (yes i can see it), but it is magical in this function and is mutating db state.